web/src/app/schedules/ScheduleOverrideCreateDialog.tsx

```diff
@@ -87,13 +80,25 @@ export default function ScheduleOverrideCreateDialog({
       subTitle={variantDetails[variant].desc}
       errors={nonFieldErrors(error)}
       notices={notices} // create and edit dialog
-      onSubmit={() => mutate()}
+      onSubmit={() =>
+        commit(
+          {
+            input: {
+              ...value,
+              scheduleID,
+            },
+          },
+          { additionalTypenames: ['UserOverrides'] },
```

Comment:
UserOverrides is not a type, the type that is returned from the `userOverrides` query is `UserOverrideConnection`

We also need `Schedule` here because an override will possibly invalidate `Schedule.shifts`

```suggestion
          { additionalTypenames: ['UserOverrideConnection', 'Schedule'] },
```

Reply:
Weird